### PR TITLE
fix file format demo-rasa.json

### PR DESCRIPTION
**Pull request by amn41**
See this issue: https://github.com/golastmile/rasa-nlu-trainer/issues/16

### CODE REVIEW

File: data/examples/rasa/demo-rasa.json [hunk 1]
```diff
@@ -1,299 +1,197 @@
...
-          {
-            "start": 8, 
-            "end": 15, 
-            "value": "chinese", 
-            "entity": "cuisine"
-          }
-        ]
       },
       {
-        "text": "show me chines restaurants",
```
Comment (tmbo) [inline data/examples/rasa/demo-rasa.json]: I think this "interesting" spelling was on purpose to test the synonyms (as the value is different from the found text). I'd either undo that change or create a better synonym example in this test data.

**Comment by tmbo** [comment]
Should rather go into 7.x shouldn't it?

**Comment by amn41** [comment]
good point

**Comment by sheldon** [comment]
Hey there

I'm new to rasa and ran into this issue while doing the tutorial. I'd strongly suggest merging up to master anyway, to avoid confusing new users :)

I eventually figured out the formatting issue and came right, but it took a good half hour of digging around the related issues & pull requests to figure it out.

If you'd like I can PR it to the appropriate place. Do you have a documented process for contributing?

Cheers
Sheldon

**Comment by tmbo** [comment]
I fixed it on master & on the 0.7.x branch.

@sheldon I am sorry for your lost time. The contribution process is rather simple, either create or look for an issue you want to work on (e.g. the ones marked with `help wanted` are usually a good starting point) and then create PR where we can discuss the change :)

If you have any further questions, let me know.

**Comment by sheldon** [comment]
That's ok, just glad it got fixed up in the end. Thanks @tmbo :)

**Pull request closed without merging**